node/src/actors/chain_manager/handlers.rs

```diff
@@ -274,6 +278,21 @@ impl Handler<GetHighestCheckpointBeacon> for ChainManager {
     }
 }
 
+/// Handler for GetHighestVrfInput message
+impl Handler<GetHighestVrfInput> for ChainManager {
+    type Result = Result<CheckpointBeacon, failure::Error>;
+
+    fn handle(&mut self, _msg: GetHighestVrfInput, _ctx: &mut Context<Self>) -> Self::Result {
+        if let Some(chain_info) = &self.chain_state.chain_info {
+            Ok(chain_info.highest_vrf_output)
+        } else {
+            log::error!("No ChainInfo loaded in ChainManager");
+
+            Err(ChainInfoError::ChainInfoNotFound.into())
+        }
+    }
+}
+
```

Comment:
The handler for `GetHighestVrfInput` is a getter for `highest_vrf_output`? I think I understand the rationale, but still sounds confusing to me :rofl:

Reply:
Good catch, initially I called the variable highest_vrf_input and thats why the handler is named like that haha. Changing ti